include/PetriEngine/Colored/Reduction/RedRuleParallelPlaces.h

```diff
@@ -0,0 +1,26 @@
+/*
+ * Authors:
+ *      Nicolaj Østerby Jensen
+ *      Jesper Adriaan van Diepen
+ *      Mathias Mehl Sørensen
+ */
+
+#ifndef VERIFYPN_REDRULEPARALLELPLACES_H
+#define VERIFYPN_REDRULEPARALLELPLACES_H
+
+#include "ReductionRule.h"
+
+namespace PetriEngine::Colored::Reduction {
+    class RedRuleParallelPlaces : public ReductionRule {
+    public:
+        std::string name() override { return "ParallelPlaces"; }
```

Comment:
We should add functionality/change this name, such that in the result csv's it has a short name such as `1`